core/includes/menu.inc

```diff
@@ -828,7 +828,7 @@ function _menu_translate(&$router_item, $map, $to_arg = FALSE) {
     $tab_parent_map = explode('/', $router_item['tab_parent']);
   }
   for ($i = 0; $i < $router_item['number_parts']; $i++) {
-    if ($link_map[$i] == '%') {
+    if ($link_map[$i] == '%' && isset($path_map[$i])) {
```

Comment:
Backdrop will throw a PHP notice if a sub-tab cannot be displayed, this suppresses this notice.